Blacklight/libraries/Forking.php

```diff
@@ -126,7 +126,7 @@ public function __construct()
         SerializableClosure::removeSecurityProvider();
         $this->colorCli = new ColorCLI();
 
-        $this->dnr_path = PHP_BINARY.' misc/update/multiprocessing/.do_not_run/switch.php "php  ';
+        $this->dnr_path = PHP_BINARY.' /var/www/nntmux/misc/update/multiprocessing/.do_not_run/switch.php "php  ';
```

Comment:
This was intentionally set like this because not everyone uses same root folder.